dev/com.ibm.websphere.appserver.features/visibility/public/acmeCA-2.0/resources/l10n/com.ibm.websphere.appserver.acmeCA-2.0.properties

```diff
@@ -0,0 +1,17 @@
+###############################################################################
+# Copyright (c) 2019 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=This feature automates interactions with a certificate authority (CA) by using the Automated Certificate Management Environment (ACME) protocol.
```

Comment:
@kristip17 @jvanhill I think this looks good from a grammar and style perspective. My only question is if the description reflects the user's goal in implementing the feature. Would a user implement this feature because they think "I need to automate interactions with a CA using ACME"? Or are they more likely to be interested in downstream functions that ACME supports? 

If there is a more direct functional benefit from using this feature, we should mention it. However, if automating CA interactions via ACME is a common user goal that should be familiar to someone implementing this feature without further elaboration, then the description is ok as is.

Reply:
Automating CA interactions via ACME is a common user goal and should be familiar to someone using this feature.

Reply:
ok- approved